src/main/java/baseball/service/RandomNumGenerator.java

```diff
@@ -0,0 +1,16 @@
+package baseball.service;
+
+import java.util.*;
+
+public class RandomNumGenerator {
+    public List<Integer> makeNum(){
+        Random random = new Random();
+        Set<Integer> set = new HashSet<>();
+
+        while(set.size()<3){
+            int num = random.nextInt(9)+1;
```

Comment:
9도 마찬가지로 의미있는 숫자이니 상수로 만들어도 좋을 것 같아요!

```suggestion
            int num = random.nextInt(9)+1;
```